frontend/src/language/en.json

```diff
@@ -0,0 +1,65 @@
+{
+    "Ongoing missions": "Ongoing missions",
+    "Upcoming missions": "Upcoming missions",
+    "Past Missions": "Past Missions",
```

Comment:
We should decide on capital letters in all header words, or just in the first word. 
I vote for capital letters in all header words, giving:

"Ongoing Missions": "Ongoing Missions",
"Upcoming Missions": "Upcoming Missions",
"Past Missions": "Past Missions"

The left and right hand side in the JSON should also be identical